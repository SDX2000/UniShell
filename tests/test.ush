cd ..
stat unishell.py
cd commands
stat __init__.py
exit 1