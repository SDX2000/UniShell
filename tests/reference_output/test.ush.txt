ERROR: (TypeError) 'module' object is not callable
ERROR: (TypeError) 'module' object is not callable
================
test.ush
================
START_DIR=/home/sandeepd/source/tools/unishell/tests


